### DISCUSSION TITLE
Why was scrollingDelay removed?

### DISCUSSION THREAD

**Discussion by mdugue** [TanStack/virtual#714] [Q&A] [answered]
Aparently v3.4.0 removes the attribute `scrollingDelay`. Why is that so? Are there any recommended alternative approaches? Thanks in advance 🙏

Related commit: https://github.com/TanStack/virtual/commit/5580232814e5ce1a4d13980fd8d0ef085a376453

**Comment by piecyk** [comment]
Hi `scrollingDelay` was controlling the delay to know if scroll was complete, not be the best name to be honest. Now as we move to scrollend in observeElementOffset it's not really needed.

**Comment by kieranm** [comment]
Hey @piecyk! I notice you're using 150ms as a fallback value.

A particular use case we have is using `isScrolling` to determine whether or not to enable scroll snapping. When using a virtualizer, scroll snapping has to be disabled when not scrolling in order to recompute snap point positions.

When using scroll snapping, Safari's inertial scrolling takes more than 150ms to settle. So this value isn't long enough. It takes around 1000ms!

With scroll snapping, safari doesn't fire the `scroll` events after the snap point has been hit, even though the inertia does continue in the background. So the scrolling appears to carry on after the scroll snap gets disabled.

I appreciate this is a niche use case but until all browsers have properly implemented scrollEnd I would suggest keeping this setting.

**Reply by piecyk** [reply, marked as answer]
@kieranm ooo true, let's add it back https://github.com/TanStack/virtual/pull/719 btw renamed it to more descriptive name like `isScrollingResetDelay`
